src/nix/verify.cc

```diff
@@ -87,7 +87,7 @@ struct CmdVerify : StorePathsCommand
                 if (!noContents) {
 
                     std::unique_ptr<AbstractHashSink> hashSink;
-                    if (info->ca == "")
+                    if (info->ca)
```

Comment:
I think the logic should flip here. `info->ca == ""` should now be `!info->ca`

Reply:
Good catch
```suggestion
                    if (!info->ca)
```